spec/ruby/core/numeric/helpers/step.rb

```diff
@@ -0,0 +1,13 @@
+class Object
+  def get_args(args_type, *args)
```

Comment:
Thinking of it, the specs should probably be refactored to remove the need for this helper method. Right now it's hard to see what exactly the specs are doing as they rely on this rather odd helper method.